src/display/annotation_storage.js

```diff
@@ -41,6 +41,17 @@ class AnnotationStorage {
     this.onSetModified = null;
     this.onResetModified = null;
     this.onAnnotationEditor = null;
+
+    if (typeof PDFJSDev === "undefined" || PDFJSDev.test("TESTING")) {
+      // For testing purposes.
+      Object.defineProperty(this, "_setValues", {
+        value: obj => {
+          for (const [key, val] of Object.entries(obj)) {
+            this.setValue(key, val);
+          }
+        },
+      });
```

Comment:
Given that this function doesn't capture any private state from the constructor it doesn't really need to be defined in this file, behind the "if testing" check. Can it be moved to `test/driver.js` (the only place where it's used), as
```js
function setAnnotationsStorage(storage, obj) {
  for (const [key, val] of Object.entries(obj)) {
    storage.setValue(key, val);
  }
}
```

Reply:
Given that we're already using this format for test-only methods in e.g. `src/display/api.js` I'd be inclined to leave this as-is here, to avoid having to move code around if we e.g. want to use this helper in future unit-tests.